plugins/woocommerce/includes/class-wc-remote-logger.php

```diff
@@ -0,0 +1,111 @@
+<?php
+declare( strict_types = 1 );
+
+use Automattic\WooCommerce\Utilities\FeaturesUtil;
+
+/**
+ * WooCommerce Remote Logger
+ *
+ * The WooCommerce remote logger class adds functionality to log WooCommerce errors remotely based on if the customer opted in and several other conditions.
+ *
+ * No personal information is logged, only error information and relevant context.
+ *
+ * @class WC_Remote_Logger
+ * @since 9.2.0
+ * @package WooCommerce\Classes
+ */
+class WC_Remote_Logger {
+	/**
+	 * Determines if remote logging is allowed based on the following conditions:
+	 *
+	 * 1. The feature flag for remote error logging is enabled.
+	 * 2. The user has opted into tracking/logging.
+	 * 3. The store is allowed to log based on the variant assignment percentage.
+	 * 4. The current WooCommerce version is the latest so we don't log errors that might have been fixed in a newer version.
+	 *
+	 * @return bool
+	 */
+	public function is_remote_logging_allowed() {
+		if ( ! FeaturesUtil::feature_is_enabled( 'remote_logging' ) ) {
+			return false;
+		}
+
+		if ( ! $this->is_user_opted_in() ) {
+			return false;
+		}
+
+		if ( ! $this->is_variant_assignment_allowed() ) {
+			return false;
+		}
+
+		if ( ! $this->is_latest_woocommerce_version() ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Check if the user has opted into tracking/logging.
+	 *
+	 * @return bool
+	 */
+	private function is_user_opted_in() {
```

Comment:
```suggestion
	private function is_tracking_opted_in() {
```

I think it might be clearer to use the word `tracking` here 🤔

Reply:
Good suggestion! Changed it to is_tracking_opted_in() in b619484107933004bb068f8e60760d62394684db